filecoin-proofs/src/api.rs

```diff
@@ -67,19 +65,93 @@ pub struct SealOutput {
     pub piece_inclusion_proofs: Vec<PieceInclusionProof<PedersenHasher>>,
 }
 
+#[derive(Clone, Debug)]
+pub struct GeneratePoStOutput {
+    pub proof: Vec<u8>,
+}
+
 /// Generates a proof-of-spacetime, returning and detected storage faults.
 /// Accepts as input a challenge seed, configuration struct, and a vector of
 /// sealed sector file-path plus CommR tuples.
-///
 pub fn generate_post(
     post_config: PoStConfig,
-    challenge_seed: ChallengeSeed,
+    challenge_seed: &ChallengeSeed,
     input_parts: Vec<(Option<String>, Commitment)>,
-) -> error::Result<GeneratePoStDynamicSectorsCountOutput> {
-    generate_post_dynamic(GeneratePoStDynamicSectorsCountInput {
-        post_config,
+    faults: &[u64],
```

Comment:
QUESTION: What does one of these `u64`s represent? An index in `input_parts`? Or a sector id?

Reply:
@dignifiedquire - Still waiting on an answer to this question :)

Reply:
sector id